tests/functional/tnf/test_tnf_deployment.py

```
@@ -0,0 +1,570 @@
+        logger.test_step("Verify Ceph health after rebalance")
+        check_ceph_health_after_add_capacity(ceph_rebalance_timeout=3600)
+
+
+@pytest.mark.polarion_id("OCS-XXXX")
+@turquoise_squad
+class TestTNFFailover:
+    """
+    TNF failover scenarios with workloads
+    """
+
+    @pytest.fixture(scope="function")
```

Ruff (0.16.2)

[warning] `scope='function'` is implied in `@pytest.fixture()`

Remove implied `scope` argument

(PT003)